### ISSUE TITLE
No way to insert ± in units

### ISSUE THREAD

**Issue by MithicSpirit** [josephwright/siunitx#699]
There appears to be no way to insert a `±` symbol in units, since `\pm` is replaced with "pm" (for picometer). `\mp` for the opposite still works, however.

**Comment by josephwright** [comment]
That's true (or at least there is no convenient way), but then I've never come across a case where it was needed: can you point to one?

**Comment by MithicSpirit** [comment]
Yeah I've pretty much never had to do it either, except for an assignment where it was convenient to use the mass of a charged pion as a unit (`\text{\ensuremath{m_{\pi^\pm}}}`).
